tests/conftest.py

```diff
@@ -238,20 +238,20 @@ def tensorflow_training_py_version(tensorflow_training_version, request):
 @pytest.fixture(scope="module", params=["py2", "py3"])
 def tensorflow_inference_py_version(tensorflow_inference_version, request):
     version = Version(tensorflow_inference_version)
-    if Version("1.15.4") <= version < Version("1.16"):
+    if version == Version("1.15") or Version("1.15.4") <= version < Version("1.16"):
         return "py36"
     return _tf_py_version(tensorflow_inference_version, request)
 
 
 def _tf_py_version(tf_version, request):
     version = Version(tf_version)
-    if Version("1.15.4") <= version < Version("1.16"):
+    if version == Version("1.15") or Version("1.15.4") <= version < Version("1.16"):
         return "py3"
     if version < Version("1.11"):
         return "py2"
-    if Version("2.0.3") <= version < Version("2.1"):
+    if version == Version("2.0") or Version("2.0.3") <= version < Version("2.1"):
```

Comment:
Why do we removing the patch/revision numbers?

Reply:
We are not removing the previous patch numbers. I changed it to make the 2.0.0, 2.0.1, and 2.0.2 versions be tested with py2, while 2.0.3 and 2.0.4, which weren't released with py2, are only tested with py3.

Moreover, "2.0" previously pointed to "2.0.1" on the `tensorflow.json` file. I have now updated it to point it to the latest patch version, which means that "2.0" cannot be tested with py2 anymore.